### PR TITLE
Added initial changelog and ran prepare package release

### PR DESCRIPTION
**Pull request by louwie17**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Added a changelog file to the create product editor block package and ran the prepare package script: https://github.com/woocommerce/woocommerce/tree/trunk/tools/package-release#prepare-packages

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Just look if things look accurate.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement

#### Comment 

</details>

### CODE REVIEW

File: packages/js/create-product-editor-block/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Changelog 
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0](https://www.npmjs.com/package/@woocommerce/create-product-editor-block/v/1.0.0) - 2023-06-28 
+
+-   Minor - Initial version of @woocommerce/create-product-editor-block [#38263]
+-   Minor - Do not add script prop to generated block.json [#38496]
+
+[See legacy changelogs for previous versions](https://github.com/woocommerce/woocommerce/blob/68581955106947918d2b17607a01bdfdf22288a9/packages/js/create-product-editor-block/CHANGELOG.md).
```
Comment (octaedro) [inline packages/js/create-product-editor-block/CHANGELOG.md]: It seems like there is something wrong with this link. It's showing 404 error

Screenshot 2023-06-28 at 13 31 05

**Review by octaedro — COMMENTED** [review]
Good job @louwie17! This looks good and is testing well here. I just left a small comment.

File: packages/js/create-product-editor-block/CHANGELOG.md (hunk 1 above)
Comment (louwie17) [reply packages/js/create-product-editor-block/CHANGELOG.md]: Good catch, I deleted it, as it is not required.

**Comment by louwie17** [comment]
> Good job @louwie17! This looks good and is testing well here. I just left a small comment.

Thanks @octaedro, could you double check, I removed the link that you referred to in your comment. It was not necessary.

**Review by octaedro — APPROVED** [review]
@louwie17 Thank you for removing the link. LGTM! :shipit:

**Pull request merged**